content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title_tag: Create Infrastructure with Pulumi Deployments Click-to-Deploy | Learn Pulumi
+title: "Create Infrastructure with Pulumi Deployments Click-to-Deploy"
+layout: single
+description: |
+    Create and deploy infrastructure in the browser with Pulumi Deployments and the New Project Wizard.
+meta_desc: In this step-by-step tutorial, learn how to create and manage cloud infrastructure using Pulumi Deployments, the New Project Wizard, and GitHub integration.
+meta_image: meta.png
+weight: 50
+summary: |
+    In this tutorial, you'll learn how to create, deploy, and manage cloud infrastructure using Pulumi Deployments with click-to-deploy. You'll start by using the New Project Wizard to set up and launch infrastructure from your browser, selecting a template to get started quickly. Additionally, you'll configure the Pulumi GitHub App to support additional deployment triggers, including push-to-deploy.
```

Comment:
I thought I called this out before, but we don't actually "configure the Pulumi GitHub App to support additional deployment triggers, including push-to-deploy". All we do is have them install the GitHub app. (Sorry to be nickpicky on this, but it always bugs me when docs like this suggest they'll cover something and then don't.)

Reply:
thx